superset/sql/dialects/firebolt.py

```diff
@@ -109,42 +109,15 @@ def _parse_unnest(self, with_alias: bool = True) -> exp.Unnest | None:
             expressions = self._parse_wrapped_csv(self._parse_expression)
             offset = self._match_pair(TokenType.WITH, TokenType.ORDINALITY)
 
-            alias = self._parse_table_alias() if with_alias else None
-
-            if alias:
-                if self.dialect.UNNEST_COLUMN_ONLY:
-                    if alias.args.get("columns"):
-                        self.raise_error("Unexpected extra column alias in unnest.")
-
-                    alias.set("columns", [alias.this])
-                    alias.set("this", None)
-
-                columns = alias.args.get("columns") or []
-                if offset and len(expressions) < len(columns):
-                    offset = columns.pop()
-
-            if not offset and self._match_pair(TokenType.WITH, TokenType.OFFSET):
-                self._match(TokenType.ALIAS)
-                offset = self._parse_id_var(
-                    any_token=False, tokens=self.UNNEST_OFFSET_ALIAS_TOKENS
-                ) or exp.to_identifier("offset")
-
```

Comment:
There's no aliased `UNNEST` in Firebolt, not `WITH`/`OFFSET` after it [[docs](https://old.docs.firebolt.io/sql-reference/commands/select.html)].